### PR TITLE
fixed members.yml. added test to prevent bad entries. updated README

### PR DESCRIPTION
**Pull request by aykamko**
@chenhoward broke `bin/setup` by putting bad entries into `db/members.yml`.
That wasn't his fault since we never tested that file. I added tests.

Also cleaned up README and some other stuff.

### CODE REVIEW

File: spec/models/member_spec.rb [hunk 1]
```diff
@@ -18,4 +18,15 @@
...
   it { should validate_presence_of :email }
   it { should validate_presence_of :member_role_id }
+
+  describe "members.yml seed file" do
+    YAML.load_file(Rails.root.join('db', 'members.yml'))['members'].each do |member_data|
+      it "should not fail to create %s %s" % [member_data['first_name'], member_data['last_name']] do
+        member_data.delete 'role'
+        member_data.key?('is_alumnus') && member_data.delete('is_alumnus')
+        member_data['member_role_id'] = 0
+        expect{ Member.create!(member_data) }.to_not raise_error
```
Comment (aykamko) [inline spec/models/member_spec.rb]: fuck you hound

**Comment by chenhoward** [comment]
Now there is an inconsistency in one of our rake tasks.  The reason I removed the emails was because of the update members task (found in libs/tasks/update.task) would create duplicates on production because they would search against first name, last name, and email, and the emails were inconsistent with the entries in the members table (they all were test@gmail.com).  The reason the original emails were removed was due to the fact that we weren't sure if members wanted their emails to be publicly available.  I think the best course of action is to update the rake task to only find_or_initialize based on the first_name and last_name parameters.  This will work until we have two people with the same name.  The more scaleable option is to publicly have their emails in our repo but we should probably ask peoples permission.  That being said if someone goes through our commit history it is there.

**Comment by aykamko** [comment]
aaah ok, got it. ill fix tomorrow

**Comment by aykamko** [comment]
@chenhoward:

about page is now served directly from the yml file. email is no longer necessary, but if it isn't provided then i show an ugly paw pictures instead... so we should probably get everyone's emails.

also, @SamLau95, idk where else the Member model is used, but if it was only useful for the about page, then it might be useful to axe that model entirely

**Comment by SamLau95** [comment]
Yup, let's axe that model (but in a separate PR). Can you make an issue so we don't forget about it?

File: README.md [hunk 2]
```diff
@@ -8,12 +8,15 @@ The Blueprint Website
...
-Run ```cd . && bin/setup``` from the project root to get everything set up.
+Setup:
+
+1. Create a `database.yml` file in `config/`.
+  - `config/database.yml.sample` is provided as an example; you'll need to tweak it a bit.
+2. Run `./bin/setup` from the project root.
 
 ## Cal Blueprint
-![bp](http://bptech.berkeley.edu/assets/logo-full-large-d6419503b443e360bc6c404a16417583.png "BP Banner")
 **[Cal Blueprint](http://www.calblueprint.org/)** is a student-run UC Berkeley organization devoted to matching the skills of its members to our desire to see social good enacted in our community. Each semester, teams of 4-5 students work closely with a non-profit to bring technological solutions to the problems they face every day.
```
Comment (SamLau95) [inline README.md]: nit: Oh, there used to be a nice banner image there that I kinda liked. Could we add it back?

**Comment by SamLau95** [comment]
Looks good to me! Feel free to merge + push after the nit is addressed. To be on the same page, we're going to want to change the about page to be completely static soon (before new mems are in) so that they can practice making a PR and deploying.

File: Gemfile [hunk 3]
```diff
@@ -9,6 +9,7 @@ gem 'figaro', '~> 1.0.0'
 gem 'sucker_punch', '~> 1.3.1'
 gem 'aasm', '~> 4.1.0'
 gem 'activesupport-json_encoder', '1.1.0'
+gem 'dalli' # memcache
```
Comment (chenhoward) [inline Gemfile]: Oh snap we now caching BP Engineering lvling up

File: Gemfile.lock [hunk 4]
```diff
@@ -483,3 +485,6 @@ DEPENDENCIES
   title (~> 0.0.5)
   turbolinks (~> 2.5.1)
   uglifier (>= 1.3.0)
+
+BUNDLED WITH
```
Comment (chenhoward) [inline Gemfile.lock]: I actually hate how we probably have to match bundler versions now

**Comment by chenhoward** [comment]
This PR is :basketball:er!

**Comment by chenhoward** [comment]
But yeah I guess feel free to merge and push once Sam's nit is addressed

File: README.md (hunk 2 above)
Comment (aykamko) [inline README.md]: @SamLau95 the link was broken :(
Would you like to suggest another one?

**Comment by aykamko** [comment]
Also, @SamLau95 @chenhoward right now like 80% of our members will have the default Blueprint paw.jpg as their pic.

I want to go get everyone's emails off of prod before deploying. Thoughts?

**Comment by SamLau95** [comment]
I see; you can just take the emails from our roster in Drive?

**Pull request merged**